Week2/six.js

```diff
@@ -0,0 +1,28 @@
+// 6.1 Declare an empty array
+
+let animals=[];
+
+// 6.2
+console.log("the value of the 'animals'will be: []");
+
+// 6.3 
+
+console.log(animals);
+
+// 6.4
+
+animals=['horse', 'lion', 'eagle'];
+
+// 6.5
+
+console.log(animals);
+
+// 6.6 
+
+animals [3] = 'baby pig';
```

Comment:
Ah, this works technically yeah, but it's cleaner/easier to use [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)

Reply:
i did not know JavaScript Array push() Method; just reviewed it ; so it would be better if i write as below;

animals=['horse', 'lion', 'eagle'];
 ["horse", "lion", "eagle"]
animals.push('baby pig');
console.log(animals);
["horse", "lion", "eagle", "baby pig"]

Reply:
Yeah that looks good! Can you update your code then I can review it again?